src/main/java/org/wso2/lsp4intellij/contributors/LSPCompletionContributor.java

```diff
@@ -52,6 +52,10 @@ public void fillCompletionVariants(CompletionParameters parameters, CompletionRe
 
     @Override
     public boolean invokeAutoPopup(@NotNull PsiElement position, char typeChar) {
+        if (!FileUtils.isFileSupported(position.getContainingFile().getVirtualFile())) {
+            return false;
```

Comment:
Shall we declare a common variable for the virtual file instance to avoid invoking `position.getContainingFile().getVirtualFile()` twice? :)

Reply:
@NipunaRanasinghe good that you pointed out, i think i have missed it after trying out the fix to do the cleanups. I think its there is no performance penalty, but from code quality point of its always good to reduce duplicates :)

Reply:
I have pushed the changes.